platform/core/build_for_android.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -e    # Exit if any command fails
+
+# ----------------------- Command options ------------------------ #
+android_version=24
+ndk_version=r22b
+build_type=debug
+release_flag=""
+
+while [ $# -gt 0 ]; do
+
+   if [[ $1 == *"--"* ]]; then
+        param="${1/--/}"
+        declare $param="$2"
+   fi
+
+  shift
+done
+
+if [[ $build_type == release ]]; then
+    release_flag="--release"
+fi
+
+# ----------------------- NDK setup ------------------------ #
+google_repo=https://dl.google.com/android/repository
+ndk_lib=android-ndk-${ndk_version}
+android_export=export/android
+ndk_dir=${android_export}/NDK
+# Override NDK_HOME to point to downloaded version.
+# Will be used by `cargo ndk` to generate the libraries
+export NDK_HOME=${ndk_dir}/${ndk_lib}
+
+if [ ! -d $NDK_HOME ]; then
+	mkdir -p ${android_export}
+
+	echo "*** Installing NDK $ndk_version"
+	ndk_download_link=$google_repo/$ndk_lib
+	os_name=""
+	case "$OSTYPE" in
+	    darwin*)
+	        os_name=darwin
+	    ;;
+	    linux*)
+	        os_name=linux
+	    ;;
+	    msys*)
+	        os_name=windows
+	    ;;
+	    cygwin*)
+	        os_name=windows
+	    ;;
+	    *)
+	        echo "unsupported OS: $OSTYPE"
+	        exit 1
+	    ;;
+	esac
+
+	ndk_download_link=$google_repo/$ndk_lib-$os_name-x86_64.zip 
+	curl -L $ndk_download_link -o ${ndk_dir}.zip
```

Comment:
You can call this file anything you want, as long as you use it throughout. Calling it `ndk_dir` is a bit confusing to me. But not a big deal.

Reply:
Yeah, now after the vacation, I see what you mean :)